.gitignore

```diff
@@ -25,12 +25,8 @@ images/google/protobuf/*.h
 .gitid
 criu/criu
 criu/unittest/unittest
-criu/arch/*/sys-exec-tbl*.c
 # x86 syscalls-table is not generated
 !criu/arch/x86/sys-exec-tbl.c
```

Comment:
Nit: while at here, this one only is needed with the line above.
So you can purge those two lines as well.